index.bs

```diff
@@ -273,73 +273,68 @@ requestFontsButton.onclick = async function() {
 
-</div>
+<aside class=note>
+These name properties are exposed to align with property usage in [[CSS-FONTS-4]], e.g. ''@font-face'', 'font-family', and so on. The [=font representation/PostScript name=] can be used as a unique key, for example when specifying a font when creating content or matching fonts against existing content. The [=font representation/full name=] or [=font representation/family name=] can be used for user-visible font selection UI, and the [=font representation/style name=] can be used to provide more specific selections.
+</aside>
 
 
 <!-- ============================================================ -->
-## Name Table ## {#concept-name-table}
+# Permissions Integration # {#permissions-integration}
 <!-- ============================================================ -->
 
-A [=/font representation=] has a <dfn for="font representation">name table</dfn>, which is the [=/font table=] in its [=font representation/table list=] with [=font table/tag=] \``name`\`.
-
-The [=font representation/name table=] has a [=/map=] <dfn for="name table">names</dfn>, which is a mapping from an {{unsigned short}} (a <dfn>name ID</dfn>) to a [=/localized string map=]. [=/Name IDs=] are defined as part of the [[!OPENTYPE]] <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/name">naming table</a> specification.
-
-A <dfn>localized string map</dfn> is a [=/map=] from a BCP 47 language tag to a {{USVString}}. [[BCP47]]
-
-Note: [[!OPENTYPE]] does not actually specify the usage of [[BCP47]] language tags; instead, platform specific language IDs are used, e.g. US English is language ID = 0x409 for Windows and language ID = 0 for Macintosh. This specification assumes a mapping exists.
+Enumeration of local fonts requires a permission to be granted.
 
-<div algorithm >
+<!-- ============================================================ -->
+## System Fonts ## {#concept-system-fonts}
+<!-- ============================================================ -->
 
-A [=/font representation=]'s <dfn for="font representation" lt="name string">name string |id| for |language|</dfn> is given by these steps:
+A <dfn>system font</dfn> is a font that is available system-wide provided by the operating system.
 
-1. Let |names| be the [=/font representation=]'s [=font representation/name table=]'s [=name table/names=].
-1. Let |name| be |names|[|id|], a [=/localized string map=].
-1. If |name|[|language|] [=map/exists=], then return it.
-1. Otherwise, return the first entry in |name|.
+To <dfn lt="read as font representation">read a system font as a font representation</dfn> is to provide a [=/font representation=] equivalent to the font, or failure otherwise. This means providing all properties of a [=/font representation=]:
```

Comment:
It'd be best to define an algorithm for each kind of font that's supported by any UA. Failing that, we should specify that the algorithm is up to the user agent. By [Hyrum's Law](https://www.hyrumslaw.com/), I suspect some websites will wind up relying on UA-specific quirks here, although maybe the number of websites that use this feature will be small enough to avoid that.

Reply:
Adding text, which is admittedly squishy. 

I'm tempted to say something non-normative that the expected consumers of the font data will use a small number of libraries built to parse common font formats, and maybe hint that we expect these libraries to evolve in conjunction with new font formats and features. But I suspect that will attract "what about....?" comments.